src/components/Contacts/ContactListTable.jsx

```diff
@@ -50,31 +54,68 @@ const ContactListTable = ({ contacts, deleteContact }) => {
   const contactsCopy = [...contacts];
   const sortedContacts = contactsCopy.sort(comparePerson);
 
+  const columnTitlesArray = [
+    {
+      field: 'Contact',
+      width: 160,
+      headerAlign: 'center',
+      align: 'center',
+      sortable: false
+    },
+    {
+      field: 'Profile',
+      renderCell: (contactData) => <ContactProfileIcon contact={contactData} />,
+      sortable: false,
+      filterable: false,
+      width: 70,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      field: 'actions',
+      type: 'actions',
+      width: 70,
+      getActions: (contactData) => [
+        <GridActionsCellItem
+          icon={<DeleteOutlineOutlinedIcon />}
+          onClick={() => deleteContact(contactData.row.Delete)}
+        />
+      ]
+    }
+  ];
+
   return (
-    <ThemeProvider theme={theme}>
-      <TableContainer component={Paper} sx={{ margin: '1rem 0', maxWidth: '500px' }}>
-        <Table aria-label="contact list table">
-          <TableHead>
-            <TableRow>
-              {columnTitlesArray.map((columnTitle) => (
-                <TableCell key={columnTitle} align="center">
-                  {columnTitle}
-                </TableCell>
-              ))}
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {sortedContacts?.map((contact) => (
-              <ContactListTableRow
-                key={contact.webId}
-                contact={contact}
-                deleteContact={deleteContact}
-              />
-            ))}
-          </TableBody>
-        </Table>
-      </TableContainer>
-    </ThemeProvider>
+    <Box sx={{ margin: '20px 0' }}>
+      <DataGrid
```

Comment:
[There are a lot of accessibility issues with this component](https://github.com/mui/mui-x/issues?q=is%3Aopen+label%3A%22component%3A+data+grid%22+label%3Aaccessibility+), but I do think that a `grid` role is the right one so don't want to stand in the way of progress on this point. I tried it with VoiceOver and it wasn't really doing what I was expecting, but don't know if that's our or their side.

Reply:
Think having a fix for those would be great. We could open up an issue to fix the accessibility problems here in a follow-up. 👍